### PR TITLE
docs (Operator):  add documentation for operator switchMap

### LINKED ISSUE #123: Add docs for operator : transformation - switchMap
Add docs for operator switchMap under group transformation 
 
Below is minimum details needed 
 
- [ ] name 
- [ ] operatorType 
- [ ] signature 
- [ ] parameters 
- [ ] short description 
- [ ] Technical description 
- [ ] Visual example  (Marble diagram) 
- [ ] Real world example (Non-framework) 
- [ ] Related Operators

### PR DESCRIPTION
**Pull request by mustafamg**
Closes #123

### CODE REVIEW

**Comment by ladyleet** [comment]
thx @mustafamg !

File: src/operator-docs/transformation/switchMap.ts [hunk 1]
```diff
@@ -1,6 +1,69 @@
...
+    that inner Observable. When a new inner Observable is emitted, <code>switchMap</code>
+    stops emitting items from the earlier-emitted inner Observable and begins
+    emitting items from the new one. It continues to behave like this for
+    subsequent inner Observables.`
+  },
+  examples: [
+    {
+      name: 'Rerun an interval Observable on every click even',
+      code: `
+      var clicks = Rx.Observable.fromEvent(document, 'click');
```
Comment (ashwin-sureshkumar) [inline src/operator-docs/transformation/switchMap.ts]: please use `const`

**Comment by ladyleet** [comment]
thx for this @mustafamg !

**Comment by ashwin-sureshkumar** [comment]
@mustafamg Thank you. Could you please update the commit message to follow guidelines. 

`docs(operators): add documentation for operator switchmap` 

Please rebase as well.

**Comment by mustafamg** [comment]
I am not expert with git stuff, @ashwin-sureshkumar Would you please advice how to rebase?

**Comment by ashwin-sureshkumar** [comment]
@mustafamg - Just incase for your reference. Please follow below steps, I am being very explicit ignore if you have already done lot of these.
```
git remote -v
```
The above should two remote addresses 

```
origin	git@github.com:mustafamg/rxjs-docs.git (fetch)
origin	git@github.com:mustafamg/rxjs-docs.git (push)
upstream	git@github.com:ReactiveX/rxjs-docs.git (fetch)
upstream	git@github.com:ReactiveX/rxjs-docs.git (push)
```

If you have only `origin` then please perform the below step 

``` 
git remote add upstream git@github.com:ReactiveX/rxjs-docs.git
```

Then perform below steps 

```
git checkout master 
git pull --ff upstream master
git checkout doc-switchmap
git rebase master -i 
/// To fix the commit message and update your commit message according to guidelines
git commit --amend 
git push -f
```

**Comment by mustafamg** [comment]
Many thanks, @ashwin-sureshkumar for the reply, it helped me do it. Thanks :)

**Comment by ladyleet** [comment]
thanks so much @ashwin-sureshkumar for this! @mustafamg sorry it took a second! holidays!

**Pull request merged**